susemanager-utils/susemanager-sls/salt/bootstrap/init.sls

```diff
@@ -230,6 +236,20 @@ salt-install-contextvars:
 
 {%- if not transactional %}
 {% include 'bootstrap/remove_traditional_stack.sls' %}
+
+mgr_update_basic_pkgs:
+  pkg.latest:
+    - pkgs:
+      - openssl
+{%- if grains['os_family'] == 'Suse' and grains['osrelease'] in ['11.3', '11.4'] and grains['cpuarch'] in ['i586', 'x86_64'] %}
+      - pmtools
+{%- elif grains['cpuarch'] in ['aarch64', 'x86_64'] %}
+      - dmidecode
+{%- endif %}
+{%- if grains['os_family'] == 'Suse' %}
+      - zypper
+{%- elif grains['os_family'] == 'RedHat' or grains['os_family'] == 'openEuler' %}
+      - yum
 {%- endif %}
```

Comment:
This is not only affecting "openEuler", could you please described what is motivating this change?

Reply:
Checking it...

Reply:
Removed. Nothing seemed to break without that part.